### ISSUE TITLE
TUI build can't find the_Foundation even though it's there

### ISSUE THREAD

**Issue by itsdeadguy** [skyjake/lagrange#614]
Hello, I've been trying to build the TUI version on Void Linux (i recursive cloned the skyjake.fi repo as per the readme).

I run the build-tui script, sealcurses gets cloned and all seems to go well but then i get this error message:

```
CMake Error at cmake/Depends.cmake:21 (find_package):
  By not providing "Findthe_Foundation.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "the_Foundation", but CMake did not find one.

  Could not find a package configuration file provided by "the_Foundation"
  (requested version 1.6.0) with any of the following names:

    the_FoundationConfig.cmake
    the_foundation-config.cmake

  Add the installation prefix of "the_Foundation" to CMAKE_PREFIX_PATH or set
  "the_Foundation_DIR" to a directory containing one of the above files.  If
  "the_Foundation" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:80 (include)
```

Scrolling up i can see that the_Foundation packages get built and installed. Shortly before the error i see this message:

```
Package the_Foundation was not found in the pkg-config search path.
Perhaps you should add the directory containing `the_Foundation.pc'
to the PKG_CONFIG_PATH environment variable
No package 'the_Foundation' found
```

**Comment by skyjake** [comment]
When you run the script, does the directory `${BUILD_DIR}/lib/cmake/the_Foundation` actually get created? Sometimes there's an extra subdirectory under `lib`, or it has a different name altogether. It could be necessary to force the_Foundation to be installed in a specific directory, not just set the install prefix.

**Comment by itsdeadguy** [comment]
the path it's actually at is `${BUILD_DIR}/lib64/cmake/the_Foundation`, notice the "64".

**Comment by skyjake** [comment]
3396bea3b0eb821542ba54b3137c4c9472f2d193 updates the TUI build script to override the default install directory name. This may help.

However, there appears to have been some regressions in the TUI build recently, so it may not work well enough for normal use at the moment.

**Comment by skyjake** [comment]
Included in v1.16.7, released today.

**Comment by itsdeadguy** [comment]
that did, thank you.
